ReactWindows/ReactNative/DevSupport/WebSocketJavaScriptExecutor.cs

```diff
@@ -190,9 +195,40 @@ public void Dispose()
 
         private async Task SendMessageAsync(int requestId, string message)
         {
-            _messageWriter.WriteString(message);
-            await _messageWriter.StoreAsync();
-            // TODO: check result of `StoreAsync()`
+            if (!_isDisposed)
+            {
+                _messageWriter.WriteString(message);
+                await _messageWriter.StoreAsync();
+                // TODO: check result of `StoreAsync()`
+            }
+            else
+            {
+                DoCallback(requestId, null);
```

Comment:
Not going to fix this, trying to catch the exception and prevent it from going all the way to the redbox is too difficult for test code.

---
In reply to: [68448798](https://github.com/ReactWindows/react-native-windows/pull/499#discussion_r68448798) [](ancestors = 68448798)